File: Pyramid Pattern/code1.c
#include <stdio.h>

void main(){

	int row ;
	printf("enter no. of rows : ");
	scanf("%d",&row);

	for(int i = 1 ; i  <= row ; i++){

		for(int a = 0; a < row -i ;a++ ){

			printf(" \t");
		}

		for(int j = 0 ; j < (2*i)-1; j++ ){


			printf("1\t");
		}
		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/8/code1.java
import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("enter no. of rows");
		int row = s.nextInt();
		int tem = 1;
		for(int i = 0; i < row ; i++){

			for(int j = 1;j <= row;j++){

				System.out.print(tem+"	");
				if(j != row){

					tem++;

				}
			}
			System.out.println("\n");
			

		}


	}
}
